tests/framework/installer/ceph_installer.go

```diff
@@ -572,8 +572,13 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(systemNamespace string, name
 		nodes, err := h.GetNodeHostnames()
 		checkError(h.T(), err, "cannot get node names")
 		for _, node := range nodes {
-			err = h.cleanupDir(node, h.hostPathToDelete)
-			logger.Infof("removing %s from node %s. err=%v", h.hostPathToDelete, node, err)
+			if h.cleanupHost {
+				err = h.verifyDirCleanup(node, h.hostPathToDelete)
```

Comment:
How does this verify the cleanup? I see that it starts a job, but where do we look at the job results to see if it succeeded?

Reply:
That only verifies that the job was creaed. Whether the job completed successfully needs more validation, but we can save that for a separate PR.